src/helpers/AngularPosition.cpp

```diff
@@ -0,0 +1,10 @@
+#include "AngularPosition.h"
+
```

Comment:
All .cpp files for helpers can we converted to .inl. This is a better way since we just have function implementations. We can do catkin_install.